RIRImageResizing/Classes/NSObject/RIRResizeImageOperationParameters.m

```diff
@@ -0,0 +1,46 @@
+//
+//  RIRResizeImageOperationParameters.m
+//  Pods
+//
+//  Created by Richard Reitzfeld on 11/21/16.
+//
+//
+
+#import "RIRResizeImageOperationParameters.h"
+
+#import <ResplendentUtilities/RUConditionalReturn.h>
+
+
+
+
+
+@implementation RIRResizeImageOperationParameters
+
+#pragma mark - NSObject
+-(instancetype)init
+{
+    kRUConditionalReturn_ReturnValueNil(YES, YES);
+    return [self init_with_newSize:CGSizeZero
+                        resizeMode:UIImage_RIRResizing_ResizeMode_none
+                             scale:0.0f];
+}
+
+#pragma mark - init
+-(instancetype)init_with_newSize:(CGSize)newSize
+                      resizeMode:(UIImage_RIRResizing_ResizeMode)resizeMode
+                           scale:(CGFloat)scale
+{
+    kRUConditionalReturn_ReturnValueNil(newSize.height <= 0 || newSize.width <= 0, YES);
+    kRUConditionalReturn_ReturnValueNil(UIImage_RIRResizing_ResizeMode__isInRange(resizeMode) == NO, YES);
+    
+    if (self = [super init])
+    {
+        _newSize = newSize;
```

Comment:
We generally want to use setter methods instead of instance variables, in case we one day wanted to be able to KVO on these properties (and other reasons, it's good convention).

Go ahead and declare these properties in the .m's class extension, as they're declared in the .h, but without `readonly`, then call the setters here.